### PR TITLE
Culled all the shit from README

### PR DESCRIPTION
**Pull request by mitchmcdee**
It always felt super intimidating to me as a newbie looking at that monolithic hunk of text when getting started, so I think it's worth culling it down to just the relevant good stuff :)

Lemme know if I've accidentally removed something important lel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+2. `dos2unix bin/hubot`
+3. `bin/hubot`
 
 ### Running uqcsbot locally
 
-**If you're having trouble building/running the bot don't be afraid to ask for help in the UQCS slack**
+**If you're having trouble building/running the bot, don't be afraid to ask for help in the UQCS slack**
 
 You can test your hubot by running the following, however some plugins will not
 behave as expected unless the [environment variables](#configuration) they rely
```
Comment (nickl93) [inline README.md]: This link will be broken when the configuration section below is removed.

Comment (nickl93) [inline README.md]: This link will be broken when the configuration section below is removed.

Comment (mitchmcdee) [reply README.md]: good catch :^) fixed

**Comment by gricey432** [comment]
Some of this longer information could be important. Instead of being removed, have you considered making a `/docs` folder and moving advanced documentation into multiple markdown files?

**Comment by TRManderson** [comment]
Pretty much none of the extra info is important and it all comes from the autogenerated readme.

**Pull request merged**